app/scripts.babel/element_observer.js

```diff
@@ -0,0 +1,41 @@
+const HANDLER_CLASS_PREFIX = 'handler_';
+
+class ElementObserver {
+  constructor() {
+    this._handlers = [];
+    this._observer = new MutationObserver(mutations => {
+      for (const mutation of mutations) {
+        for (const node of Array.from(mutation.addedNodes)) {
+          if (!(node instanceof Element)) {
+            continue;
+          }
+
+          this._handlers.forEach((item, i) => {
+            if (node.matches(item.selector) &&
+                !node.classList.contains(HANDLER_CLASS_PREFIX + i)) {
+              // console.log('online ' + item.selector);
+              item.handler(node);
+            }
```

Comment:
The problem is that some elements can be found among the existing elements (returned by `document.querySelectorAll`) and in the same time passed to the `MutationObserver` callback as recently added. Specifying of the CSS class prevents these elements from being processed twice. However, if the element wasn't found among the existing elements, it can't be processed twice, so there's no need to add the CSS class.